src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerDefinitionCommands/New-QuotaObject.ps1

```diff
@@ -61,6 +67,14 @@ function New-QuotaObject
         [int32]
         $VmScaleSetCount,
 
+        [Parameter(Mandatory = $false)]
+        [int32]
+		$MaxAllocationStandardManagedDisksAndSnapshots,
```

Comment:
Please shorten these.

Reply:
This has been discussed and PM has opened a bug to track this and get done.

Reply:
You can change it here, it won't affect server side.  Please consult with PM to change it now in PowerShell so we don't have a breaking change later.

Reply:
StandardManagedDiskAndSnapshotSize